### PR TITLE
Named other resources

### PR DESCRIPTION
**Pull request by mssteuer**
Adding url_based_resource type and changing additional resources to object with both name and URL in order to have named resources (and display named anchors rather than just URLs)

Signed-off-by: Michael Steuer <michael@make.software>

### CODE REVIEW

**Comment by ihor** [comment]
LGTM

**Review by mrkara — APPROVED** [review]
Looks good to me.

**Review by piotrwitek — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The [JSON Schema](schema.json) provided in this project is compatible with
 [JSON schema Draft 2019-09](https://json-schema.org/specification-links.html#2019-09-formerly-known-as-draft-8)
 and can be validated with compatible schema validators and IDEs. 
 The provided template [account-info.json](account-info.json) implements the schema and can be validated against 
-it using online tools such as [JsonSchemaValidator.net](https://www.jsonschemavalidator.net/s/OWGua5Rt).
+it using online tools such as [JsonSchemaValidator.net](https://www.jsonschemavalidator.net/s/TYCUEe8S).
```
Comment (piotrwitek) [inline README.md:23]: @mssteuer  the new schema validator link is missing a new rule added, cannot test if working correctly because of it.

**Review by piotrwitek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (piotrwitek) [reply README.md:23]: Sorry, I found it, turned out search doesn't work correctly on that web page.

Comment (piotrwitek) [reply README.md:23]: Looks good!

**Review by piotrwitek — APPROVED** [review]

**Review by ihor — APPROVED** [review]

**Pull request merged**